src/components/post/Post.js

```diff
@@ -109,9 +110,9 @@ class Post extends Component {
                   alignItems="flex-start"
                 >
                   <Grid className="viewContainer" className={classes.viewCount}>
-                    {upload.metadata.views == null
-                      ? "0"
-                      : upload.metadata.views}
+                    {upload
```

Comment:
as far as I know, views is not returned after an upload ... kinda curious how your testing went on this one

Reply:
just tested it... it doesn't work

Reply:
then it will return 0 for upload

Reply:
for it to work, please change it back to the way it was